docs/api/transaction/get-transaction-events.example.json

```diff
@@ -0,0 +1,67 @@
+{
+  "limit": 96,
+  "offset": 0,
+  "events": [
```

Comment:
Please change this example to be a real response instead of using mock data ("0x1234", etc.)

Reply:
Is this a real response? If so, why are the `event_index` entries out of order for the same tx?